bin/docker-run.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash 
 
-# Print commands and exit on errors
-set -ex 
```

Comment:
Removing `x` here since the next command prints a bunch of stuff that we don't need to see related to creating a commit hash.